autometa/taxonomy/ncbi.py

```diff
@@ -364,27 +332,66 @@ def parse_merged(self):
             logger.debug('merged loaded')
         return merged
 
-    def is_common_ancestor(self, parent_taxid, child_taxid):
-        """Determines whether the provided taxids have a non-root common ancestor
+    def is_common_ancestor(self, taxid1, taxid2):
```

Comment:
Is there a different name we can use other than numbers? I do not think this is an issue, but I do not think it is common to assign variables numeric values.